optaplanner-docs/src/main/asciidoc/LocalSearch/LocalSearch-chapter.adoc

```diff
@@ -283,15 +283,14 @@ Advanced configuration:
 [[tabuSearchAlgorithm]]
 === Algorithm Description
 
-Tabu Search works like Hill Climbing, but it maintains a tabu list to avoid getting stuck in local optima.
+Tabu Search is a Local Search that maintains a tabu list to avoid getting stuck in local optima.
```

Comment:
This change is not related to PLANNER-1593.

Reply:
Indeed, request from JustinG that I am lumping in this already small PR. For small PR's I think it's fine to combine such things (it's a separate commit for history of course).